src/ui/components/filters/filterboxcomponent.js

```diff
@@ -217,15 +217,15 @@ export default class FilterBoxComponent extends Component {
       this._filterComponents.push(component);
       this._filterNodes[i] = component.getFilterNode();
     }
-    this._saveFilterNodesToStorage();
+    this._saveFilterNodesToStorage(true);
```

Comment:
The `FilterBox` component is not only use for Facets. It can be used to represent a collection of static filters as well. Do we want this behavior on that case too? Or should we guard this with `this.config.isDynamic`?